modules/broker-core/src/main/java/io/ballerina/messaging/broker/core/Message.java

```diff
@@ -176,6 +178,17 @@ public boolean hasContent() {
         return Objects.nonNull(metadata);
     }
 
+    /**
+     * Get expire timestamp
+     *
+     * @return get expire timestamp as a {@link java.lang.Long}
+     */
+    public long getExpiryTimestamp() {
+        FieldValue expiryFieldValue = metadata.getProperties().getValue(Metadata.EXPIRATION);
+        ShortString expiryTime = (ShortString) expiryFieldValue.getValue();
```

Comment:
Can expiryFieldValue be null?

Reply:
No. But expiryTime can be empty. Fixed the PR.